src/utils.js

```diff
@@ -5,11 +5,26 @@ import {readFile as readFileCB} from 'fs';
 var readFile = Promise.promisify(readFileCB);
 
 /* Helper functions */
-const basename = filepath => path.basename(filepath, path.extname(filepath));
-const dirname  = filepath => path.normalize(path.dirname(filepath));
-const parentDirname = filepath => dirname(filepath).split(path.sep).pop();
-const removeNumbers = str  => str.replace(/^[0-9|\.\-]+/, '');
-const getFiles = glob => globby(glob, {nodir: true });
+
+function basename (filepath) {
+  return path.basename(filepath, path.extname(filepath));
+}
+function dirname (filepath) {
+  return path.normalize(path.dirname(filepath));
+}
+function parentDirname (filepath) {
+  return dirname(filepath).split(path.sep).pop();
+}
+function removeLeadingNumbers (str) {
+  return str.replace(/^[0-9|\.\-]+/, '');
```

Comment:
Can we add `@example`s to these functions to demonstrate the I/O?

Also, for `removeLeadingNumbers`, can we broaden the delimiters it will catch? I'm not sure what the original Fabricator did, but I would expect all of these to match:

![screen shot 2016-03-04 at 2 36 30 pm](https://cloud.githubusercontent.com/assets/61204/13542310/baed3466-e216-11e5-846a-2b555af99d3f.png)

Here's another pattern that might work:

![screen shot 2016-03-04 at 2 44 06 pm](https://cloud.githubusercontent.com/assets/61204/13542425/94fdf578-e217-11e5-9d64-6cab5399f3d1.png)